share/eds/p2p/client.go

```diff
@@ -132,12 +135,8 @@ func (c *Client) doRequest(
 		// no eds was returned, but the request was valid and should be retried
 		return nil, nil
 	case p2p_pb.Status_INVALID:
-		return nil, fmt.Errorf("request for root %s marked as invalid by peer", dataHash.String())
+		fallthrough
 	default:
-		return nil, fmt.Errorf("peer responded with unknown status %s", resp.GetStatus())
+		return nil, fmt.Errorf("request status for root %s marked as invalid", dataHash.String())
```

Comment:
can you still add details re the resp.GetStatus as well?

Reply:
done